README.md

```diff
@@ -218,6 +226,26 @@ await ws_test.run()
 assert ws_test.is_complete()
 ```
 
+### Using rest requests
+Attaching simple rest get request and sending it:
+```py
+rest_request = (
+    RestRequest("https://example.com", "GET")
+    .with_body({"some_key": some_value})
+)
+
+ws_test = (
+    WSTest("wss://example.com")
+    .with_request(rest_request)
+)
+
+await ws_test.run()
+
+respone = ws_tester.received_request_responses[0]
```

Comment:
I think this would be a nicer example if we iterated over each response in the request responses and printed it rather than extracting the first response

```py
for response in ws_tester.received_request_responses:
    print(response.status_code)
    print(response.json())
````